app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -27,6 +27,11 @@ class EventsFragment : Fragment() {
     private val eventsViewModel by viewModel<EventsViewModel>()
     private lateinit var rootView: View
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        eventsRecyclerAdapter.setEventLayout(EventUtils.EVENTS)
```

Comment:
@iamareebjamal Here is one case

Reply:
Use them locally then, no need for these in EventUtils

Reply:
Okay :+1:

Reply:
Did the changes :+1: